mlir/test/Dialect/Linalg/vectorize-convolution.mlir

```diff
@@ -61,6 +61,35 @@ func.func @conv1d_nwc_4x2x8_memref(%input: memref<4x6x3xf32>, %filter: memref<1x
 
 // -----
 
+// This test is same as above but for i1 type.
```

Comment:
[nit] It's worth pointing out that the only difference should be `kind` and that's the main that's checked.

Also, I would add "check" lines for `kind` to the original tests (i.e. `@conv1d_nwc_4x2x8_memref` and `@conv1d_ncw_4x8x2_memref_i1`).